qcodes/instrument/base.py

```diff
@@ -628,3 +628,12 @@ def __getstate__(self):
             'were trying to use a local instrument (defined with '
             'server_name=None) in a background Loop. Local instruments can '
             'only be used in Loops with background=False.')
+
+    def validate_status(self, verbose=0):
+        """ Validate the values of all gettable parameters """
```

Comment:
a) Cool,  just write the assumption in the docstring ! 

b) In this function 2 and 1 are the same then, which I don't like, and I don't see it documented or understandable from the code. 
Change to true and false in this  case.